libnmstate/nm/plugin.py

```diff
@@ -140,6 +146,21 @@ def get_dns_client_config(self):
     def refresh_content(self):
         self._ctx.refresh_content()
 
+    def _check_version_mismatch(self):
+        nm_client_version = _nm_client_decode_version(self._ctx.client)
```

Comment:
`nm_client_version = self._ctx.client.get_version()` is enough. No need to wrap one line code as function.

Reply:
You are right, thanks!